src/components/ui/AntDesign.jsx

```diff
@@ -0,0 +1,267 @@
+                                />
+                            </List.Item>
+                            <List.Item>
+                                <List.Item.Meta 
+                                        avatar={<InfoCircleFilled/>}
+                                        title={props.data.action.activity.title}
+                                />
+                            </List.Item>
+                        </List>
+                        <Link>Edit section</Link>
+                    </Col>
+                </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={12}>
+                    <Space align="center">
+                        <InfoCircleFilled/>
+                        <Title level={3}>Sign-ups</Title>
+                    </Space>
+                </Col>
+                <Col span={12}>
+                    <QuestionCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">Click in list to book</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <InfoCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">No Pending sign-ups to be booked</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Contact person</Title>
+                        </Space>
+                        <Space>
+                            <Avatar shape="square" size={50}>
+                                {props.data.person.first_name.charAt(0)}
+                                {props.data.person.last_name.charAt(0)}
+                            </Avatar>
+                            <Space direction="vertical">
+                                <Title level={4}>
+                                    {props.data.person.first_name} {props.data.person.last_name}
+                                </Title>
+                                <Text type="secondary">Drag another person here or select one to change</Text>
+                            </Space>
+                        </Space>
+                    </Space>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Booked participants</Title>
+                        </Space>
+                        <Cascader
+                            action={props.data.action}
+                            onChange={onChange}
+                            placeholder="Please select"
+                            showSearch={{ filter }}
```

Comment:
This is cool! But I'm wondering if you could add some dummy data to filter, because right now the list is empty.